tdp/cli/commands/deploy.py

```diff
@@ -98,16 +106,53 @@ def deploy(
         check_services_cleanliness(cluster_variables)
 
         operation_runner = OperationRunner(dag, ansible_executor, cluster_variables)
+        if resume:
+            if sources or targets:
+                raise click.BadParameter(
+                    f"sources or targets resume can not be used resume"
+                )
+            resumed_deployment = get_deployment(session_class, resume)
+            if resumed_deployment.operations[-1].state == "Success":
```

Comment:
You should check if `operations` has any deployment before accessing the last statement